gallery/src/styles/NxPageLocalHeader/NxPageLocalHeaderFullExample.tsx

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {
+  NxGlobalSidebarNavigation,
+  NxGlobalSidebarNavigationLink,
+  NxPageSidebar,
+  NxButton,
+  NxFontAwesomeIcon,
+  NxBackButton,
+  NxStatefulGlobalSidebar,
+  NxPageMain
+} from '@sonatype/react-shared-components';
+import { faArrowLeft, faArrowRight, faLink, faEdit } from '@fortawesome/free-solid-svg-icons';
+
+const sidebarLogoPath = require('../../assets/logo-plaid-villain-text.png');
+
+export default function NxPageLocalHeaderFullExample() {
+  return (
+    <>
+      <aside className="nx-page-local-header">
+        <NxBackButton href="#/pages/nx-page-local-header" targetPageTitle="Documentation" />
+        <div className="nx-page-local-header__actions">
```

Comment:
Not `btn-bar`?

Reply:
> Not .btn-bar?

I guess I wasn't confident that it would only contain buttons, at also it has different styling from a usual `.nx-btn-bar` due to the left border and surrounding spacing.